pages/vi/vi-faq.md

```diff
@@ -26,63 +26,116 @@
 
 ### General Internship Questions
 
-#### Q1: What will I be working on during this internship?
```

Comment:
Removing `####` in front of questions makes us unable to quick "jump" to respective questions when we click on questions on the top of the page